api/src/main/resources/org/openmrs/module/ugandaemrreports/reports/MER_TX_CURRENT_2019.json

```diff
@@ -0,0 +1,635 @@
+    "type": "summary",
+    "measure": "https://ugisl.mets.or.ug/Measure/mer-tx-indicators",
+    "reporter": {
+        "reference": "Organisation/MK"
+    },
+    "period": {
+        "start": "#parameter.startDate|YYYY-MM-dd#",
+        "end": "#parameter.endDate|YYYY-MM-dd#"
+    },
+    "group": [
+        {
+            "code": {
+                "coding": [
+                    {
+                        "code": "ADX_TX_CURR_AGE_GRP_SEX"
+                    }
+                ]
+            },
+            "measureScore": {
+                "value": #TOTAL#
+            },
+            "stratifier": [
+                {
+                    "code": [
+                        {
+                            "coding": [
+                                {
+                                    "code": "165816005",
+                                    "display": "TX_CURR disaggregated by AGE GROUP and SEX"
+                                }
+                            ]
+                        }
+                    ],
+                    "stratum": [
+                        {
+                            "value": {
+                                "coding": [
+                                    {
+                                        "code": "P0Y--P1Y:F",
+                                        "display": "< 1 year Female"
+                                    }
+                                ]
+                            },
+                            "measureScore": {
+                                "value": #2a#
+                            }
+                        },
+                        {
+                            "value": {
+                                "coding": [
+                                    {
+                                        "code": "P0Y--P1Y:M",
+                                        "display": "<1 year Male"
+                                    }
+                                ]
+                            },
+                            "measureScore": {
+                                "value": #2b#
+                            }
+                        },
+                        {
+                            "value": {
+                                "coding": [
+                                    {
+                                        "code": "P1Y--P5Y:F",
+                                        "display": "1­-4 years Female"
+                                    }
+                                ]
+                            },
+                            "measureScore": {
+                                "value": #3a#
+                            }
+                        },
+                        {
+                            "value": {
+                                "coding": [
+                                    {
+                                        "code": "P1Y--P5Y:M",
+                                        "display": "1­-4 years Male"
```

Comment:
@dbaluku strange text "SHY" in the dispay before the dash

Reply:
Removed